src/app/shared/components/navigation/navigation.component.html

```diff
@@ -4,9 +4,4 @@ <h1>Captain Cook</h1>
 	<button (click)="toggleMenu()"><img src="../../../../assets/icons/hamburger.svg" alt=""></button>
 </div>
 </header>
-<div [ngClass]="{active: isOpen}" class="menu-items">
-	<ul>
-		<li class="menu-item"><a [routerLink]="['/planner']">Planner</a></li>
-		<li class="menu-item"><a [routerLink]="['/recipes']">Recipes</a></li>
-	</ul>
-</div>
+<app-dropdown [ngClass]="{close: isOpen}"></app-dropdown>
```

Comment:
I would like the dropdown to be able to function as a separate component. Right now its workings are dependent on the navigation.